velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1041,4 +1041,31 @@ struct ParseDateTimeFunction {
   }
 };
 
+template <typename T>
+struct TimeZoneMinuteFunction : public TimestampWithTimezoneSupport<T> {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(
```

Comment:
This probably should share the same implementation with #4520

Reply:
@Yuhta Can we push this separately for this once. For future functions will implement the similar ones in the same PR.

Reply:
@Ann-Rose once #4520 lands, let's refactor and re-use the code to implement this function.